akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -383,8 +402,33 @@ object Sink {
    * to use a bounded mailbox with zero `mailbox-push-timeout-time` or use a rate
    * limiting stage in front of this `Sink`.
    */
+  @deprecated("Use `actorRef` that takes onFailureMessage instead. It allows controling the message that will be sent to the actor on failure.", since = "2.5.10")
```

Comment:
I'm not sure we should deprecate them? It still makes sense (is easier to use) for untyped actors? We can have both.

Reply:
I am also on the fence about this one. I would like to keep the API surface as small as possible. The more general `actorRef` methods on the untyped side were not requested by anyone so far. So maybe a better approach would be to keep them private.

Reply:
good point, we can at least start out by having them private